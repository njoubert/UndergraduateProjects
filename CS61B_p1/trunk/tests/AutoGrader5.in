Alex lives around here.
Peter lives in the underground house.
What do you know about Alex?
What do you know about Peter?

