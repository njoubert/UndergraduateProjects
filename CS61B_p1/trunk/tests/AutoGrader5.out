1. Alex lives around here.
2. Peter lives in the underground house.
Q: What do you know about Alex?
A: Nothing.
Q: What do you know about Peter?
A: Peter lives in the underground house.
